manager/manager.go

```diff
@@ -478,6 +478,18 @@ func (m *manager) createContainer(containerName string) error {
 	// Start the container's housekeeping.
 	cont.Start()
 
+	newEvent := &events.Event{
```

Comment:
We don't want to do this here since we will create an even each time we restart cAdvisor which is not what we want. We should do this only in detectSubcontainers() or watchForNewContainers(). In detectSubcontainers() we should also only do it if the container creation time was before the cAdvisor start time. Lets start with just watchForNewContainers() and move to detectSubcontainers() in a followup PR.